src/interfaces/packagingSObjects.ts

```diff
@@ -321,6 +321,15 @@ export namespace PackagingSObjects {
     MinPackageVersionId: string;
   };
 
+  export type InstalledPackage = {
```

Comment:
@WillieRuemmele this structure does not look like the [public docs](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_installedpackage.htm).

The query that is using this looks like it is get data from relationship. I think this type belongs in `[packagingInterfacesAndType.ts](https://github.com/forcedotcom/packaging/blob/wr/packageInstalledList/src/interfaces/packagingInterfacesAndType.ts)`